requirements.in

```diff
@@ -14,3 +14,6 @@ langdetect
 django-silk
 honcho
 flower
+polyglot
+pyicu
+pycld2
```

Comment:
Not for this PR, but we should consider [pipenv](https://pipenv.readthedocs.io/en/latest/), it's much nicer to use than virtualenv+pip-tools.